### PR TITLE
Violette lepercq/director of machine learning insights part 4

### PR DESCRIPTION
**Pull request by VioletteLepercq**

### CODE REVIEW

File: ml-director-insights-4.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+</style>
+<h1>Director of Machine Learning Insights [Part 4]</h1>
+
+<div class="blog-metadata">
+    <small>Published November 25, 2022.</small>
+</div>
+</head>
+<body>
+
+_If you're interested in building ML solutions faster visit [hf.co/support](https://huggingface.co/support?utm_source=article&utm_medium=blog&utm_campaign=ml_director_insights_3) today!_
```
Comment (BrigitteTousi) [inline ml-director-insights-4.md:21]: add comma after visit

File: ml-director-insights-4.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+
+👋 Welcome back to our Director of ML Insights Series! If you missed earlier Editions you can find them here:
+
+- [Director of Machine Learning Insights [Part 1]](https://huggingface.co/blog/ml-director-insights)
+- [Director of Machine Learning Insights [Part 2 : SaaS Edition]](https://huggingface.co/blog/ml-director-insights-2)
+- [Director of Machine Learning Insights [Part 3 : Finance Edition]](https://huggingface.co/blog/ml-director-insights-3)
+
+🚀 In this fourth installment, you’ll hear what the following top Machine Learning Directors say about Machine Learning’s impact on their prospective industries: Javier Mansilla, Shaun Gittens, Samuel Franklin, and Evan Castle. —All are currently Directors of Machine Learning with rich field insights.
```
Comment (BrigitteTousi) [inline ml-director-insights-4.md:30]: replace prospective by respective

File: ml-director-insights-4.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+**Fun Fact:** I love reading science fiction, and my idea of retirement includes resuming the teenage dream of writing short stories.📚
+
+**Mercado Libre:** The biggest company in Latam and the eCommerce & fintech omnipresent solution for the continent
+ 
+#### **1. How has ML made a positive impact on e-commerce?**
+I would say that ML made possible the impossible in a few cases (like fraud prevention in general) and in the vast majority of the others, optimized processes and flows in ways we wouldn't imagine.
```
Comment (BrigitteTousi) [inline ml-director-insights-4.md:45]: I suggest modifying to: "made the impossible possible in specific cases like fraud prevention and optimized processes and flows in ways we couldn't have imagined in a vast majority of other areas."

File: ml-director-insights-4.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+In the middle, there are applications where ML enabled a next-level of UX that otherwise would be very expensive (but maybe possible). For example, the discovery and serendipity added to users' journey navigating between listings and offers.
+
+We ran search, recommendations, ads, credit-scoring, moderations, forecasting of several key aspects, logistics, and lot’s more core units with Machine Learning optimizing at least one of its fundamental metrics.
```
Comment (BrigitteTousi) [inline ml-director-insights-4.md:49]: replace lot's with "a lot"

File: ml-director-insights-4.md [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+  
+For instance, starting complex instead of with the simplest baseline possible. For instance not measuring the with/without machine learning impact. For instance, investing in tech without having a clear clue of the boundaries of the expected gain.
+
+Last but not least: thinking only in the short term, forgetting about the hidden impacts, technical debts, maintenance, and so on.
+
+
+#### **4. What excites you most about the future of ML?**
+Talking from the perspective of being on the trench crafting technology with our bare hands like we used to do ten years ago, definitely what I like the most is to see that we as an industry are solving most of the slow, repetitive and boring pieces of the challenge.
+
+It’s of course an ever-moving target, and new difficulties arose.
```
Comment (BrigitteTousi) [inline ml-director-insights-4.md:73]: replace arose with arise

File: ml-director-insights-4.md [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+But we are getting better to incorporate mature tools and practices that will lead to shorter cycles of model-building, which at the end of the day reduces time to market.
```
Comment (BrigitteTousi) [inline ml-director-insights-4.md:74]: replace to incorporate with "at incorporating" 
Add comma after which
Add comma after end of the day

File: ml-director-insights-4.md [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+2. Another critical challenge regarding ML in engineering is that the field is mainly categorized by the need for problem-solving, which often requires creativity.  As of now, few great cases exist today of ML agents being truly “creative” and capable of “thinking out-of-the-box” since current ML solutions tend to result merely from a search through all possible solutions.  In my humble opinion, though a great many solutions can be found via these methods, ML will have somewhat of a ceiling in engineering until the former can consistently demonstrate creativity in a variety of problem spaces.  That said, that ceiling is still pretty high, and there is much left to be accomplished in ML applications in engineering.
+
+
+ 
+#### **3. What’s a common mistake you see people make trying to integrate ML into Engineering?**
```
Comment (BrigitteTousi) [inline ml-director-insights-4.md:104]: add "when" before trying

Comment (VioletteLepercq) [reply ml-director-insights-4.md]: done 👌

File: ml-director-insights-4.md (hunk 6 above)
Comment (VioletteLepercq) [reply ml-director-insights-4.md]: done 👌

File: ml-director-insights-4.md (hunk 5 above)
Comment (VioletteLepercq) [reply ml-director-insights-4.md]: done 👌

File: ml-director-insights-4.md (hunk 4 above)
Comment (VioletteLepercq) [reply ml-director-insights-4.md]: done 👌

File: ml-director-insights-4.md (hunk 3 above)
Comment (VioletteLepercq) [reply ml-director-insights-4.md]: done 👌

File: ml-director-insights-4.md (hunk 2 above)
Comment (VioletteLepercq) [reply ml-director-insights-4.md]: done 👌

File: ml-director-insights-4.md (hunk 1 above)
Comment (VioletteLepercq) [reply ml-director-insights-4.md]: done 👌

**Pull request merged**